db/seeds.rb

```diff
@@ -7,30 +7,26 @@
-Video.create(title: "Futurama", description: "Accidentally frozen, pizza-deliverer Fry wakes up 1,000 years in the future.", small_cover_url: "/tmp/futurama.jpg", large_cover_url: "/tmp/monk_large.jpg", category_id: 2)
-Video.create(title: "Monk", description: "After the unsolved murder of his wife, Adrian Monk develops obsessive-compulsive disorder, which includes his terror of germs and contamination.", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg", category_id: 2)
-Video.create(title: "Futurama", description: "Accidentally frozen, pizza-deliverer Fry wakes up 1,000 years in the future.", small_cover_url: "/tmp/futurama.jpg", large_cover_url: "/tmp/monk_large.jpg", category_id: 2)
-Video.create(title: "Monk", description: "After the unsolved murder of his wife, Adrian Monk develops obsessive-compulsive disorder, which includes his terror of germs and contamination.", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg", category_id: 2)
-Video.create(title: "Futurama", description: "Accidentally frozen, pizza-deliverer Fry wakes up 1,000 years in the future.", small_cover_url: "/tmp/futurama.jpg", large_cover_url: "/tmp/monk_large.jpg", category_id: 2)
-monk = Video.create(title: "Monk", description: "After the unsolved murder of his wife, Adrian Monk develops obsessive-compulsive disorder, which includes his terror of germs and contamination.", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg", category_id: 2)
-Video.create(title: "Futurama", description: "Accidentally frozen, pizza-deliverer Fry wakes up 1,000 years in the future.", small_cover_url: "/tmp/futurama.jpg", large_cover_url: "/tmp/monk_large.jpg", category_id: 2)
-Video.create(title: "Monk", description: "After the unsolved murder of his wife, Adrian Monk develops obsessive-compulsive disorder, which includes his terror of germs and contamination.", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg", category_id: 2)
-Video.create(title: "Futurama", description: "Accidentally frozen, pizza-deliverer Fry wakes up 1,000 years in the future.", small_cover_url: "/tmp/futurama.jpg", large_cover_url: "/tmp/monk_large.jpg", category_id: 2)
+Video.create(title: "Futurama", description: "Accidentally frozen, pizza-deliverer Fry wakes up 1,000 years in the future.", small_cover_url: "/tmp/futurama.jpg", large_cover_url: "/tmp/monk_large.jpg", category: comedies)
+Video.create(title: "Monk", description: "After the unsolved murder of his wife, Adrian Monk develops obsessive-compulsive disorder, which includes his terror of germs and contamination.", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg", category: dramas)
+Video.create(title: "Futurama", description: "Accidentally frozen, pizza-deliverer Fry wakes up 1,000 years in the future.", small_cover_url: "/tmp/futurama.jpg", large_cover_url: "/tmp/monk_large.jpg", category: comedies)
```

Comment:
These lines are hard to read - I have to scroll to right all the time. I would recommend style you deleted on lines 13 - 17.

```ruby
Video.create!(title: "Monk",
              description: "After the unsolved murder of his wife, Adrian Monk develops obsessive-compulsive disorder, which includes his terror of germs and contamination.",
              small_cover_url: "/tmp/monk.jpg",
              large_cover_url: "/tmp/monk_large.jpg",
              category: drama)
```
and `create!` method to pop validation error if there is any.